app/assets/javascripts/services/miq_db_backup_service.js

```diff
@@ -58,6 +64,24 @@ ManageIQ.angular.app.service('miqDBBackupService', function() {
            (this.isModelValueNil(value));
   };
 
+  this.swiftSecurityProtocolNotSelected = function(model) {
+    return model.security_protocol === '' || model.security_protocol === undefined;
```

Comment:
This function seems to be never used.

Reply:
Just in case it's used. This is equal to `return !!model.security_protocol` if we don't mind `model.security_protocol !== null` is included as well.